tcmur_cmd_handler.c

```diff
@@ -747,6 +865,9 @@ static int tcmur_cmd_handler(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	case WRITE_VERIFY_16:
 		ret = handle_write_verify(dev, cmd);
 		break;
+	case WRITE_SAME_16:
```

Comment:
Yes, I think this could be done in the following patches. For now I only test the _16 case.